assignments/array-methods.js

```diff
@@ -57,30 +57,39 @@ const runners = [
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
-let fullNames = [runners];
-runners.forEach(runners => {
-  // console.log(runners);
-});
+let fullName= [];
+runners.forEach(function(__, index, arr){
+  fullName.push('${arr[index].first_Name} ${arr,[index].last_Name}')
```

Comment:
this has syntax error, did it work

Reply:
It didnt show me a syntax error
I double checked it, It didn't log the expected result